src/irc/core/irc-servers-setup.c

```diff
@@ -30,6 +30,29 @@
 #include "irc-servers.h"
 #include "sasl.h"
 
+#include <errno.h>
+
+static void read_password_file(char *str, char **password)
+{
+	char **values = g_strsplit(str, ":", -1);
+	char *path;
+	GError *err = NULL;
+
+	path = g_strdup(values[1]);
```

Comment:
The `path = g_strdup_printf()` below replaces it without freeing the old pointer.